snuba/cli/devserver.py

```diff
@@ -137,6 +137,23 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
         ),
     ]
 
+    if settings.ENABLE_DEV_FEATURES:
+        daemons += [
+            (
+                "metrics-consumer",
+                [
+                    "snuba",
+                    "multistorage-consumer",
```

Comment:
Should I use a multi-storage consumer here, or rather multiple consumers with different consumer groups?

Reply:
A multi storage consumer is the right option. The topic is the same, there is no need for multiple consumers in dev.